packages/tools/src/eventListeners/mouse/mouseDownListener.ts

```diff
@@ -102,8 +134,30 @@ function mouseDownListener(evt: MouseEvent) {
   // Prevent CornerstoneToolsMouseMove while mouse is down
   state.element.removeEventListener('mousemove', mouseMoveListener);
 
-  const startPoints = getMouseEventPoints(evt, state.element);
-  const deltaPoints = _getDeltaPoints(startPoints, startPoints);
+  state.startPoints = getMouseEventPoints(evt, state.element);
```

Comment:
we were doing `copyPoints` previously, so I'm wondering not copying will result some bug due to reference manipulation